ci/nightly/pipeline.template.yml

```diff
@@ -235,6 +235,18 @@ steps:
               args: [--redpanda]
         skip: "Disabled due to taking too long for the value provided"
 
+      # TODO(def-) Remove this when old upsert implementation is removed
+      - id: testdrive-old-upsert
+        label: ":racing_car: testdrive with old Upsert"
+        depends_on: build-aarch64
+        timeout_in_minutes: 180
+        agents:
+          queue: hetzner-aarch64-8cpu-16gb
+        plugins:
+          - ./ci/plugins/mzcompose:
+              composition: testdrive
+              args: [--system-param=storage_use_reclock_v2=false]
```

Comment:
I believe this and other mentions of `storage_use_reclock_v2` should be `storage_use_continual_feedback_upsert`

Reply:
Yes, I don't know why I keep getting them confused. :D